docker/docker-compose.yml

```
@@ -1,17 +1,17 @@
 ﻿version: "3.7"
 services:
   mongodb:
-    image: mongo:4.2.3
+    image: mongo:4.2
```

```suggestion
    image: mongo:4.4
```
i'm out of the loop, but from catching up on slack it seems like the servers are on 4.4 now, so maybe this could be too?

---

the production and dev servers are on 4.4 indeed

---

I've applied the change. Thanks @svensven!